debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -490,6 +504,9 @@ protected void handleInsert(Event event) throws InterruptedException {
             logger.debug("Skipping previously processed row event: {}", event);
             return;
         }
+        if (ignoreDmlEventByGtidSource) {
+            return;
```

Comment:
👍 . Btw, how do I enable the debug logging? I am pretty new to java -- most of my development is in PHP.